scripts/install-worker.sh

```diff
@@ -113,7 +112,7 @@ sudo systemctl restart sshd.service
 ### iptables ###################################################################
 ################################################################################
 sudo mkdir -p /etc/eks
-sudo mv $TEMPLATE_DIR/iptables-restore.service /etc/eks/iptables-restore.service
+sudo mv $WORKING_DIR/iptables-restore.service /etc/eks/iptables-restore.service
```

Comment:
fyi, I created a separated PR for prevent word split issue #1244 

need to ensure there's no `space` included in path `WORKING_DIR`

Reply:
actually, when I'm working on #1244, I did not aware of the existence of this PR. 😄